src/plugins/README.md

```diff
@@ -165,6 +165,10 @@ Log/Send out all changes to configuration to:
 - [syslog](syslog/)
 - [logchange](logchange/) prints the change of every key on the console
 
+Notification of key changes:
+
+- [internalnotification](internalnotification/) automatically update integer variables when a key is changed from inside an application
```

Comment:
keep the short description more generic. I think "automatically update registered variables"